simple_certmanager/models.py

```diff
@@ -1,5 +1,7 @@
 from datetime import datetime
+from io import StringIO
 
+from django.core.files.base import File
```

Comment:
```suggestion
from django.core.files import File
```
I *think* the parent module already exports it